packages/eslint-config/index.js

```diff
@@ -1,7 +1,13 @@
 // This is the ESLint configuation used by Redwood projects.
+import { getConfig } from '@redwoodjs/internal'
+
+const config = getConfig()
 
 module.exports = {
-  extends: './shared.js',
+  extends: [
+    './shared.js',
+    config.web.a11y ? 'plugin:jsx-a11y/recommended' : undefined,
```

Comment:
Similarly, not convinced that this is the best way to conditionally add something to an array?

Reply:
I think what you're doing is fine, but a couple of other options are:

```
  extends: config.web.a11y ? ['plugin:jsx-a11y/recommended', './shared.js'] : ['./shared.js],
```

```
  extends: [
    './shared.js',
    ...(config.web.a11y ? ['plugin:jsx-a11y/recommended'] : [],
  ]
```

Reply:
My goto for these is filtering via Boolean.

```js
extends: [
    './shared.js',
    config.web.a11y && 'plugin:jsx-a11y/recommended',
].filter(Boolean)
```